src/classes/LVL_LevelAssignBATCH_TEST.cls

```diff
@@ -629,4 +629,175 @@ private class LVL_LevelAssignBATCH_TEST {
         }
     }
 
+    /*********************************************************************************************************
+    @description
+        Execute Level Assign Batch on Contacts having Source Field value outside of all levels
+    verify:
+        The Level Field is changed to blank.
+        The Previous Level Field is populated with the Level Field as it was before the batch has been run.
+    **********************************************************************************************************/  
+    static testMethod void previousLevelShouldBePopulatedWhenCurrentLevelIsChangedToBlank() {
+        Level__c[] levels = new Level__c[] {
```

Comment:
any reason you didn't use the helper createTestLevels(), like most of the other tests do?